patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+## Title
+
+InnerSource Hackathon
+
+## Patlet
+
+In a company, initially only InnerSource enthusiasts are interested and practicing InnerSource during the early stages of InnerSource adoption; not all engineering teams are willing or have enough time and resources to adopt InnerSource. In this scenario, it is good to provide a safe space to try and adopt InnerSource through an InnerSource Hackathon event within the company.
+
+## Problem
+
+The company wants to adopt InnerSource as software development methodology but only those familiar with open source principles or those who understand the benefits of InnerSource, adopt it. It results in just a handful of InnerSource projects and is difficult to scale beyond that.
```

Comment:
Would you do that purely as a Hackthon targeted at learning about InnerSource best practices? 

I'm asking because reading your suggested pattern, made me remember some of the very early InnerSource days at my employer: There was a yearly hackathon established already. Participating teams decided to try out InnerSource best practices there early on - come to think of it, that was one of the occasions where best practices spread across team boundaries.

Reply:
Yes, the goal is to make the engineering community learn about InnerSource best practices. When ISPO or sometimes OSPO (driving InnerSource as well) is not able to reach the entire engineering community, then such an event would help draw some attention. 

Just out of curiosity, the yearly hackathon that you mentioned - was it specific to InnerSource? If so, can we quote it as a known instance?

Reply:
The one I mentioned wasn't specific to InnerSource - it had been established long before the InnerSource initiative.

What happened was that some Hackathon teams would naturally adopt InnerSource for those few days to try it out. As a result the practice was adopted by new teams afterwards.

I think this shows just how strong a Hackathon as a pattern is: Even if it is not focused 100% on spreading the practice of InnerSource it still helps teams adopt those new ways of working. We could extend the pattern such that there are two Hackathon options - those organised by ISPO/OSPO and 100% geared towards InnerSource exercises plus more generic Hackathons than we can of course include my employer as well.